tmpfiles.sh

```diff
@@ -55,6 +69,10 @@ _chattr() {
 		*) attr="+${attr}" ;;
 	esac
 	local IFS=
+	if ! owned_by_root $1 ; then
```

Comment:
need to quote `"$1"`

comes up below too

Reply:
Changed in commit 4f4576a6781fb64a6c798d131f8309802c2f9c3a